src/chrome/chromeDebugAdapter.ts

```diff
@@ -392,8 +392,13 @@ export abstract class ChromeDebugAdapter implements IDebugAdapter {
             });
         });
         this.chrome.Debugger.onResumed(() => this.onResumed());
+
+        const onScriptParsedEventName = 'target/notification/onScriptParsed';
+        this._batchTelemetryReporter.configureEvent(onScriptParsedEventName, {
+            batchCap: 50
```

Comment:
If the truncation is based on property size of 1KB, would it make more sense to add all the events we can fit in under 1 KB, and then drop events or do something smart if we detect that a property is going over 1KB if we keep adding events?

Reply:
We should do a trade-off between efficiency and quality here. We are talking about CPU cycles on customers machine here. I don't want to spend cycles on customer machine do complicated calculations or producing obscure code and reducing maintainability. And the end result is that we merely manage to salvage <10% more data. Does it really make a difference? I doubt that.

Reply:
Isn't it better to have a limit of say 100 and drop everything after that? Any reason we think that sample size will not be good sample set to get the distribution?

Reply:
We have data like

```
VS.BPT.Diagnostics.ChromeDebugAapter.aggregated.startTime	["1522092607233","1522092607391"]
```
This is a good example of a maximum length each data element can get, which is 15 bytes on average. 100 of them will reach 1500 KB, way over 1KB.

```
Will we be able to process the information if the property with 100 values gets cut because it exceeds 1 KB?
```

No, if a property carries 100 values and gets cut, it become invalid. There is no reliable way to recover these 100 values. Potentially we can recover a part of it but it's tricky, merely adding complexity to the downstream processing.

Reply:
If the point of discussion is how big the number should be, I would recommend discussing it offline. We can easily change it to whatever value. It does not have to block other code change in this PR.